test/transcoding/IntelFPGAFunctionAttributes.ll

```diff
@@ -7,7 +7,10 @@
 ;;     intel::num_simd_work_items(8),
 ;;     intel::stall_enable,
 ;;     intel::scheduler_target_fmax_mhz(1000),
-;;     intel::loop_fuse_independent(3)]] void operator()() {}
+;;     intel::loop_fuse_independent(3),
+;;     intel::component_ii(10),
+;;     intel::component_max_concurrency(12),
+;;     intel::disable_component_pipelining]] void operator()() {}
```

Comment:
These attributes have another names in clang. Could you please set actual names?

Reply:
Ah, and it does look like the medata name has also changed.